components/SpotCard.js

```diff
@@ -90,3 +121,11 @@ const Button = styled.button`
     background-color: #a0a0a0;
   }
 `;
+
+const BookmarkButton = styled.button`
+  margin: 0px;
+  border: 1px solid;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: row-reverse;
```

Comment:
You could think about working with absolute positioning to place your bookmark button. 🤔